src/Apps/ArtQuiz/Components/ArtQuizResultsTabs.tsx

```diff
@@ -22,6 +28,16 @@ export const ArtQuizResultsTabs: FC = () => {
     })
   }
 
+  const calculatePerArtworkRecommendationsLimit = useCallback(() => {
+    if (savedQuizArtworksCount <= 1) return 100
+    if (savedQuizArtworksCount <= 3) return 8
+    return 4
```

Comment:
100, 8, or 4?

Reply:
100 is no upper limit.

Reply:
Ah OK. Might be helpful to inline this explanation in a comment:

```
If user likes only 1 artwork, show ALL related
1 liked = all related
If user likes less than 3 artworks, show [8] per liked
2 liked = 16 artworks
3 liked = 24 artworks
If user likes more than 3 artworks, show [4] per liked
4 liked = 16 artworks
5 liked = 20 artworks
….
16 liked = 64 artworks
```

Reply:
Could also extract this as a function and then have the tests explain the requirements!

Reply:
I question the decision to extract or export a function exclusively for the sake of testing. Have heard arguments for and against, and still not sure where I land.

To your point though, @jonallured, I should test this component. But that sounds like after lunch Laura's problem.

🥪

Reply:
The reason: extracted functions are extremely easy to test. Its a matter of dependency injection; rather than having to mock out the world, you simply mock out the function variable inputs. Done and done.

Reply:
(Would also appreciate a comment here to avoid having to maintain magic numbers lacking context)

Reply:
Have a ticket open in the art quiz epic to flesh out the related test suite. Will address this there.